tutorials/examples/preprocess_imgs.py

```diff
@@ -0,0 +1,92 @@
+"""
+This script illustrates how to preprocess experimental CryoEM images
+before performing reconstructing 3D map.
+"""
+
+import logging
+import matplotlib.pyplot as plt
+
+from aspire.source.relion import RelionSource
+from aspire.estimation.noise import WhiteNoiseEstimator
+
+logger = logging.getLogger('aspire')
+
+starfile_in = '/tigress/junchaox/CryoEMdata/empiar10028/shiny_2sets.star'
+data_folder = '/tigress/junchaox/CryoEMdata/empiar10028/'
+pixel_size = 1.34
+num_imgs = 1024
+
+logger.info('This script illustrates how to preprocess experimental CryoEM images')
+logger.info(f'Read in images from {starfile_in} and preprocess the images')
+source = RelionSource(
```

Comment:
Let's make this a `Simulation` instead. Right now, the script will only run if the user has access to the paths above, which is unlikely to be the case for the average user.

Reply:
Switch to use simulation data as the example in commit 49330e9.